source/aggregation/stages.txt

```diff
@@ -0,0 +1,345 @@
+.. _csharp-aggregation-stages:
+
+==================
+Aggregation Stages
+==================
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: dotnet, code example, transform, pipeline
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+On this page, you can learn how to create an aggregation pipeline and add stages to it
+by using methods in the {+driver-short+}.
```

Comment:
S: since this guide is more about creating the stages than adding them to a pipeline
```suggestion
On this page, you can learn how to create an aggregation pipeline and pipeline stages
by using methods in the {+driver-short+}.
```